### ISSUE TITLE
Notice Error Accessing Empty Impressions Object

### ISSUE THREAD

**Issue by timnolte** [google/site-kit-wp#1167]
## Bug Description

When debugging is turned on I see the following notice reported:
```
Notice: Trying to get property 'impressions' of non-object in /srv/bindings/ac0e354ce5fe4a8db049e57763fa0f7e/code/web/wp-content/plugins/google-site-kit/includes/Modules/Search_Console.php on line 402
```

## Steps to reproduce

1. Go to the page editor and you'll see this flash on the screen before the Gutenberg editor takes over the screen.

## Additional Context

 - PHP Version: 7.3
 - OS: Ubuntu 19.10
 - Browser: Google Chrome 80
 - Plugin Version: 1.2.0
 - Device: Desktop

It is possible that this bug is fixed in a more recent version but I haven't updated the plugin just yet. However, when searching open issues I'm not seeing that this error has ever been reported.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

## QA Brief

* 

## Changelog entry

*

**Comment by aaemnnosttv** [comment]
Thanks for the report! This should be fixed as of 1.3.0. Can you please try with the latest release and let us know if you continue to see any notices?

**Comment by timnolte** [comment]
Confirmed this is fixed in the most recent release.
